assets/src/scripts/components/Versions.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import {
+  Badge,
+  Card,
+  ListGroup,
+  OverlayTrigger,
+  Tooltip,
+} from "react-bootstrap";
+import { VersionT } from "../types";
+
+interface VersionProps {
+  versions: VersionT[];
+}
+
+function Version({ version }: { version: VersionT }) {
+  const createdAt = new Date(version.created_at);
+  const userTimeZone =
+    Intl?.DateTimeFormat()?.resolvedOptions()?.timeZone || "UTC";
+  const userLang = navigator?.language || "en-GB";
+
+  const createdDate = createdAt.toLocaleDateString(userLang, {
+    day: "numeric",
+    month: "short",
+    year: "numeric",
+    timeZone: userTimeZone,
+  });
+
+  const createdTime = createdAt.toLocaleTimeString(userLang, {
+    hour: "2-digit",
+    minute: "2-digit",
+    timeZone: userTimeZone,
+  });
+
+  return (
+    <ListGroup.Item
+      as="li"
+      key={version.tag_or_hash}
+      variant={version.current ? "secondary" : undefined}
+    >
+      <div className="d-flex justify-content-between align-items-center">
+        <span className="d-block text-monospace font-weight-bold">
+          {version.current ? (
+            version.tag_or_hash
+          ) : (
+            <a href={encodeURI(version.url)}>{version.tag_or_hash}</a>
+          )}
+        </span>
+        {version.status === "draft" ? (
+          <Badge variant="light">Draft</Badge>
+        ) : null}
+        {version.status === "under review" ? (
+          <Badge variant="info">Under review</Badge>
+        ) : null}
+        {version.status === "published" ? (
+          <Badge variant="success">Published</Badge>
+        ) : null}
+      </div>
+      <span className="created d-block p-0">
+        <OverlayTrigger
+          placement="right"
+          overlay={
+            <Tooltip id={version.tag_or_hash}>
+              When this codelist version was added to OpenCodelists
+            </Tooltip>
+          }
+        >
+          <abbr className="font-weight-bold">Created</abbr>
```

Comment:
Bit of a nit, but the `abbr` tag in the version.html file also has the `title` attribute, which causes this behaviour for published codelists:
![image](https://github.com/user-attachments/assets/ed12b971-c033-4496-8104-c31aedb25ee5)

and this for draft codelists:
![image](https://github.com/user-attachments/assets/f5abbbbd-c98c-4ad1-bd1b-de48e30353fb)

I don't mind which we do, but probably good to have the same behaviour for both.